apps/statuses/index.coffee

```diff
@@ -0,0 +1,18 @@
+app = module.exports = require('express')()
+
+statuses = [
```

Comment:
Just what I was developing against. So I can change the url to always fail or succeed and make sure states work as they should.